### PR TITLE
Add note for preload default value change

### PR DESCRIPTION
**Pull request by beaufortfrancois**
I'd like to merge this when Chrome 62 reaches beta channel.

It is related to https://github.com/GoogleChrome/samples/pull/539

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
I'll update `wf_updated_on` later before merging.

File: src/content/en/fundamentals/media/fast-playback-with-video-preload.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ has actually been fetched.
 
 Setting the `preload` attribute to `metadata` indicates that the user is not
 expected to need the video, but that fetching its metadata (dimensions, track
-list, duration, and so on) is desirable.
+list, duration, and so on) is desirable. Note that starting in Chrome 62, the
+default value for `preload` is `metadata`. (It was `auto` previously).
```
Comment (mounirlamouri) [inline src/content/en/fundamentals/media/fast-playback-with-video-preload.md]: The feature is still disabled by default in Chrome 62. We are going to run an experiment to see the impact on users first.

**Comment by beaufortfrancois** [comment]
It has been delayed to Chrome 63 (at least).
Let's postpone until then.

**Comment by st420nd** [comment]
https://kb.mailchimp.com/?_ga=2.160112125.1313076106.1507822544-804961427.1507822544

**Comment by petele** [comment]
I *think* this is going out in Chrome 64...

**Comment by beaufortfrancois** [comment]
@petele Thank you for the reminder!
I've updated the patch and resolved conflicts. Do you mind pushing it live?

**Comment by petele** [comment]
@beaufortfrancois - i missed this this morning because it still had the `DO NOT MERGE` tag on it, it'll get pushed live tomorrow.

**Pull request merged**